ocs_ci/ocs/jenkins.py

```diff
@@ -0,0 +1,221 @@
+"""
+Jenkins Class to run jenkins specific tests
+"""
+import logging
+import re
+
+from ocs_ci.ocs.exceptions import (
+    CommandFailed, ResourceWrongStatusException, UnexpectedBehaviour
+)
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility import templating
+from ocs_ci.ocs.resources.ocs import OCS
+from subprocess import CalledProcessError
+from tests.helpers import create_pvc
+from ocs_ci.ocs.resources.pod import get_pod_obj
+from tests.helpers import wait_for_resource_state
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+
+
+log = logging.getLogger(__name__)
+
+
+class Jenkins(object):
+    """
+    Workload operation using Jenkins
+    """
+    def __init__(self):
+        """
+        Initializer function
+
+        """
+        self.ocp = OCP(namespace=constants.JENKINS_NAMESPACE)
```

Comment:
shouldn't we init OCP only after creating the Jenkins namespace?

Reply:
I created an instance, and I used it after creating the project (name space)

Reply:
but NS creation is done in this class (https://github.com/red-hat-storage/ocs-ci/pull/2313/files#diff-446d62e0a6d017722a2c711fc35cec1dR49) which means that it will be available for creation only after class initialization

Reply:
I created "Jenkins" instance before the test procedure (fixture)
tests/e2e/workloads/jenkins/test_jenkins.py